plugins/ui/src/js/src/elements/IllustratedMessage.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {
+  IllustratedMessage as DHCAIllustratedMessage,
+  IllustratedMessageProps as DHCAIllustratedMessageProps,
```

Comment:
Unsure what the A in DHCA stands for here? Should just be DHC:
```suggestion
  IllustratedMessage as DHCIllustratedMessage,
  IllustratedMessageProps as DHCIllustratedMessageProps,
```